tensor_genn/algorithms/relu_ann.py

```diff
@@ -10,18 +10,67 @@ def __init__(self,neuron_resting_voltage=-60.0,neuron_threshold_voltage=-55.0,
         membrane_capacitance=1.0, model_timestep=1.0, single_example_time=350.):
         self.Vres = neuron_resting_voltage
         self.Vthr = neuron_threshold_voltage
-        self.Cm = membrane_capacitance
+        self.Cm = membrane_capacitance # For DNN - 1.0, CNN - 0.4
         self.timestep = model_timestep
         self.single_example_time = single_example_time
 
+    def create_weight_matrices(self, tf_model):
+        tf_weights = tf_model.get_weights()
+        tf_layers = tf_model.layers
+
+        g_model_weights = []
+        n_units = [np.prod(tf_layers[0].input_shape[1:])]
```

Comment:
a comment describing what this does would be great - it might be a little obtuse to those not well versed in numpy :smile: